src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -46,28 +46,24 @@ const styles = StyleSheet.create({
     backgroundColor: COLORS.anon_primary
   },
   textContainer: {
-    paddingLeft: BUBBLE_LEFT_PADDING,
+    paddingLeft: BUBBLE_LEFT_PADDING + BUBBLE_LEFT_PADDING,
     paddingRight: BUBBLE_RIGHT_PADDING,
     paddingTop: 4,
     paddingBottom: 4,
-    borderRadius: 8,
-    borderTopEndRadius: 0,
+    borderRadius: 12,
     flex: 1
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The padding on the left of the text container has been doubled. If this is intentional and doesn't cause any layout issues, then it's fine. However, if it was a mistake, you should revert it to the original value.
```diff
- paddingLeft: BUBBLE_LEFT_PADDING + BUBBLE_LEFT_PADDING,
+ paddingLeft: BUBBLE_LEFT_PADDING,
```
Also, the border radius has been increased from 8 to 12. Ensure that this change aligns with your design guidelines.



<!-- This is an auto-generated comment by OSS CodeRabbit -->